Formula/neovim.rb

```diff
@@ -21,6 +21,7 @@ class Neovim < Formula
   depends_on "luajit"
   depends_on "msgpack"
   depends_on "unibilium"
+  depends_on "utf8proc"
```

Comment:
Scope the dependency inside a `head do` block

Reply:
@fxcoudert done